.github/workflows/lint-and-test.yml

```diff
@@ -62,6 +63,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: ${{ matrix.coverage == 'yes' }}
+      - name: Upgrade dependencies
```

Comment:
- To distinguish, this step from the one called "Install FlexMeasures & dependencies for tests", maybe we should call the former "Install FlexMeasures & (pinned) dependencies for tests"

Reply:
Resolving this as I'm moving the CI change into another PR.